working-groups/wg-stmt-summary.md

```diff
@@ -0,0 +1,20 @@
+# Statement Summary Working Group
+
+## Goals
+
+Discussion for the designing of statement summary tables in TiDB.
+
+Now we have a statement summary table [events\_statements\_summary\_by\_digest](https://pingcap.com/docs/stable/reference/performance/statement-summary/), and more features are being added to it. But we also have some details to be discussed.
+
+## Meetings
+
+Currently, we don't have an open meeting because there's only one developer in this working group.
```

Comment:
Could you share an open google doc to record the weekly progress summary? It helps others to know what's happening and get involved easily.

Reply:
Addressed.